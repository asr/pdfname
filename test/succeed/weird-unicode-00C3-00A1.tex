\documentclass{article}
\usepackage{hyperref}

% Weird Unicode `Ã¡` (U+00C3 and % U+00A1).

\hypersetup{%
  pdfauthor={SuÃ¡rez},%
  pdftitle={SuÃ¡rez},%
  pdfcreationdate={D:2017},%
}

\begin{document}
pdfname test.
\end{document}
