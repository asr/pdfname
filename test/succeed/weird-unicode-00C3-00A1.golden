ret > ExitSuccess
out > The full path name will be /tmp/suarez-2017.suarez.pdf
out >
